themes/digital.gov/src/scss/new/_uswds-overrides.scss

```diff
@@ -56,3 +56,11 @@
     mask-image: url("../img/svg-icons/material-icon-inventory_2.svg");
   }
 }
+
+// overrides usa footer until 3.8.0 for uswds comes out and refactor footer into the uswds footer component.
+// turing off sylelint due to it flagging needing to be in BEM format
```

Comment:
Stylelint was saying it needed to be BEM but I didnt see way to make that happen with this situation.